bash_command_line_101/bash_command_line_101.md

```diff
@@ -173,28 +173,43 @@ Bash, or shell, scripting is a way to interface with your computer's operating s
 
 If you are new to bash scripting, it can take some adjustment to navigate by typing text. Once you get the hang of a few basic commands, this can be an easier way to deal with files and folders than dropdown menus and dragging icons from place to place. Even the language we use is a little different. In bash scripting you will hear folders referred to as **directories**.
 
-To enter a command into the shell, type it in where the blinking cursor appears and press `Enter` or `return` on your keyboard.
+When you first open a shell, you will see a [**Command Prompt**](https://en.wikipedia.org/wiki/Command-line_interface#Command_prompt). This may look different for everyone, for example the two authors of this module see different things. One of us sees:
```

Comment:
@drelliche what you're showing here includes the MAC addresses of your computers (the alphanumeric name CHOP gives its macbooks at least is the MAC address).  Not sure of the security implications but might be good to suppress in future?